### ISSUE TITLE
When hashtag # present in SMILES, error type report as EntityRef: expecting ';' [23]

### ISSUE THREAD

**Issue by zhssakura** [ropensci/webchem#269]
Hello, 

Thanks for creating webchem. I found that when a hashtag symbol present in the SMILES, the following error type will occur:

```r
comp1 = "CC(C)CC(C)(O)C#CC(C)(O)CC(C)C"

cir_query(comp1, representation = 'smiles', resolver = NULL, first = TRUE, choices = NULL, verbose = TRUE)
#> Error in cir_query(comp1, representation = "smiles", resolver = NULL, : could not find function "cir_query"
```

Any suggestion would be helpful. Thank you.

Best wishes,

Shan

**Comment by stitam** [comment]
Thanks @zhssakura for raising this issue. The error occurs because of lack of url encoding, thanks for bringing our attention to this, we'll soon fix the function. In the meantime, if you substitute `#` in your queries with `%23` it should work.

```
comp2 <- "CC(C)CC(C)(O)C%23CC(C)(O)CC(C)C"
cir_query(comp2, representation = "inchikey")
$`CC(C)CC(C)(O)C%23CC(C)(O)CC(C)C`
[1] "InChIKey=LXOFYPKXCSULTL-UHFFFAOYNA-N"
```

**Comment by zhssakura** [comment]
Thanks @stitam,
I would try it as you suggested.
Best,
Shan

**Comment by Aariq** [comment]
closed with PR #270.  Install the development version, @zhssakura, for a version without this bug.  Thanks again for pointing this out.
